client/scss/utilities/_variables.scss

```diff
@@ -1,12 +1,156 @@
-$font-sans: 'helvetica neue', helvetica, arial, sans-serif;
-$font-mono: 'source code pro', courier, monospace;
+$base-font-size: 16; // Used for calculating ems from px;
 
-$font-xsmall: 1.3rem;
-$font-small: 1.6rem;
-$font-medium: 2rem;
-$font-large: 2.6rem;
-$font-xlarge: 3.4rem;
-$font-xxlarge: 4rem;
+$wellcome-bold: 'Wellcome Bold', 'Arial Black', Gadget, sans-serif;
+$helvetica-neue-light: 'Helvetica Neue LT W01_41488878', 'HelveticaNeue-Light', 'Helvetica Neue Light', 'Helvetica Neue', Helvetica, Arial, sans-serif; 
+$helvetica-neue-medium: 'Helvetica Neue LT W01_65 Md', 'HelveticaNeue-Medium', 'Helvetica Neue Medium', 'Arial Black', Gadget, sans-serif;
+$lettera-regular: 'Lettera Regular', 'Courier New', Courier, Monospace;
+
+//N.B. The letter-spacing value is the tracking value provided from Photoshop
+//The @font mixin which utilises this map, converts this to the correct CSS em value. 
+$fonts: (
```

Comment:
How much of a bad idea would it be to create each of these as a class (e.g. `.wb1`, `.wb2` etc.) and add them in the markup, rather than redeclaring all of the font styles everywhere we use them?

It might save a considerable amount of css? Is it too fragile?